zigbee2mqtt/config.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Zigbee2mqtt",
-  "version": "1.18.1-1",
+  "version": "1.18.1-2",
```

Comment:
Can you please update to 1.18.1-3? I've just merged another PR for ezsp adapter.